src/connection/mod.rs

```diff
@@ -2,6 +2,7 @@ extern crate dbus;
 
 use std;
 use general::*;
+use std::env;
```

Comment:
Should we `use` `std` stuff before `use`ing internal modules? i.e. move this above `use general::*;`